HelpSource/Guides/UsingMIDI.schelp

```diff
@@ -30,6 +30,20 @@ link::Classes/MIDIFunc:: has a number of convenience methods allowing you to reg
 
 See link::#Playing notes on your MIDI keyboard:: below for a simple example using the note-on and note-off MIDIFuncs.
 
+subsection:: MIDIFunc and MIDIdef: Filtering based on device or message data
+
+MIDIFunc and MIDIdef can filter incoming messages, responding to specific devices, MIDI channels or data values. For example, you may want one MIDIFunc to handle controller 1, while a different MIDIFunc handles controller 7.
+
+Filters are set by the argument immediately following the response function: code::MIDIFunc.incomingType({ ... function ... }, msgNum, chan, srcID):: where the meaning of msgNum depends on the message type.
```

Comment:
consider changing `... function ...` to `/* function */`. the first could lead to misunderstandings about whether what's inside the braces is meant literally or not. i just saw this the other day on a C++ forum, about almost exactly the same ambiguity.

Reply:
Good idea -- done.